test/unit/AutoPlay.spec.ts

```diff
@@ -48,6 +48,42 @@ describe("AutoPlay", () => {
     expect(nextStub.calledOnce).to.be.true;
   });
 
+  it("should apply animationDuration to animation when moving panel", async () => {
+    // Given
+    const plugin = new AutoPlay({ direction: "NEXT", duration: 500, animationDuration: 200 });
+    const wrapper = sandbox("flick0");
+    const viewportEl = document.createElement("div");
+    viewportEl.innerHTML = `
+      <div class="flicking-camera">
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+        <div style="width: 200px; height: 200px;"><p></p></div>
+      </div>
+    `;
+    wrapper.appendChild(viewportEl);
+    const flicking = new Flicking(viewportEl);
+    const moveStartSpy = sinon.spy();
+    const moveEndSpy = sinon.spy();
+    flicking.on(EVENTS.MOVE_START, moveStartSpy);
+    flicking.on(EVENTS.MOVE_END, moveEndSpy);
+
+    // When
+    flicking.addPlugins(plugin);
+    await waitEvent(flicking, "ready");
+
+    // Then
+    expect(moveStartSpy.called).to.be.false;
+    expect(moveEndSpy.called).to.be.false;
+    tick(600);
+    expect(moveStartSpy.calledOnce).to.be.true;
+    expect(moveEndSpy.called).to.be.false;
+    tick(500);
+    expect(moveStartSpy.calledOnce).to.be.true;
+    expect(moveEndSpy.calledOnce).to.be.true;
+    cleanup();
```

Comment:
I'm not asking you to change this part, but as AutoPlay calls Flicking's `next` internally, you can rather spy on flicking's next function and check what it's called with.
```js
const nextSpy = sinon.spy(flicking, "next");

expect(nextSpy.firstCall.calledWith(200)).to.be.true;
```

Reply:
I think this will simplify the same test process, and changed within applying the review.